### DISCUSSION TITLE
GraphQL API authentication question

### DISCUSSION THREAD

**Discussion by MarkoSchuetz** [community/community#67110] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I have an organization as owner with public repositories created by GitHub Classroom and private projects. On my user account I have the GraphQL API Explorer as an Authorized OAuth App. When I run a query that, say, returns the authors of issues I get the desired result. But when I run a query into the projectsV2, I cannot even get the title and get an error instead:

````
{
  "data": {
    "repository": {
      "projectsV2": {
        "edges": [
          null
        ]
      }
    }
  },
  "errors": [
    {
      "type": "FORBIDDEN",
      "path": [
        "repository",
        "projectsV2",
        "edges",
        0
      ],
      "extensions": {
        "saml_failure": false
      },
      "locations": [
        {
          "line": 4,
          "column": 7
        }
      ],
      "message": "Although you appear to have the correct authorization credentials, the `uprm-inso4116-2023-2024-S1` organization has enabled OAuth App access restrictions, meaning that data access to third-parties is limited. For more information on these restrictions, including how to enable this app, visit https://docs.github.com/articles/restricting-access-to-your-organization-s-data/"
    }
  ]
}
````

So, where else would I have to set the access for the GitHub GraphQL API Explorer?

**Comment by MarkoSchuetz** [comment]
Just as I had posted this I found out how to do it and I'll keep the answer here just in case someone comes here with the same question.

- Go to your personal settings
- Navigate to Applications > [Authorized OAuth Apps](https://github.com/settings/applications)
- Click on "GraphQL API Explorer"
- Scroll down to "Organization access" and grant access to that organization

It's actually very straightforward, but I had an looking for it for quite a while.

**Reply by lee-alderdice-mgh** [reply]
Thanks so much for posting this.

**Reply by CallmeT-ty** [reply]
Man, thank you. That was hidden to far
